src/components/NcListItem/NcListItem.vue

```diff
@@ -767,12 +781,34 @@ export default {
 			}
 		}
 	}
+	.list-item-content__details {
+		display: flex;
+		flex-direction: row;
+		justify-content: end;
+	}
+	&--one-line {
+		height: 50px;
+		padding: 10px;
+		margin: 2px;
+		.list-item-content__main {
+			display: flex;
+			justify-content: start;
+			gap: 12px;
+			min-width: 0;
+		}
+		.list-item-content__details {
+			display: flex;
+			flex-direction: row;
+			justify-content: end;
+		}
+	}
 
 	&__anchor {
 		display: flex;
-		flex: 1 0 auto;
+		flex: 1 0 10%;
```

Comment:
This `10%` looks a bit suspicious. But I haven't found we we use it...

Reply:
i changed it back to auto, and nothing was broken, so maybe we dont need to 10% anymore.